pages/processes/favourite.tsx

```diff
@@ -1,32 +1,42 @@
 import React, { useState } from "react";
-import styles from "./FrontPage.module.scss";
+
+import ActiveFilterSection from "components/Labels/ActiveFilterSection";
+import FilterLabelButton from "components/Labels/FilterLabelButton";
+import FilterUserButton from "components/FilterUserButton";
+import FrontPageBody from "components/FrontPageBody";
 import Head from "next/head";
 import { Layouts } from "../../layouts/LayoutWrapper";
+import { SearchField } from "components/SearchField";
 import SideNavBar from "components/SideNavBar";
-import FrontPageBody from "components/FrontPageBody";
-import { useQuery } from "react-query";
+import { SortSelect } from "../../components/SortSelect";
+import { Typography } from "@equinor/eds-core-react";
 import { getProjects } from "../../services/projectApi";
+import styles from "./FrontPage.module.scss";
+import { useQuery } from "react-query";
 import { useRouter } from "next/router";
-import { Typography } from "@equinor/eds-core-react";
-import { SortSelect } from "../../components/SortSelect";
-import { SearchField } from "components/SearchField";
-import FilterLabelButton from "components/Labels/FilterLabelButton";
-import ActiveFilterSection from "components/Labels/ActiveFilterSection";
 
 export default function FavoriteProcesses(): JSX.Element {
   const [page, setPage] = useState(1);
   const itemsPerPage = 16;
 
   const router = useRouter();
-
+  const requiredUsers = router.query?.user
+    ?.toString()
+    .split(",")
+    .map((user) => parseInt(user));
+  const requiredLabels = router.query?.rl
+    ?.toString()
+    .split(",")
+    .map((label) => parseInt(label));
```

Comment:
Consider creating creating one/two helper function(s) and reuse this on the three pages